### DISCUSSION TITLE
Disable Statusline in neo-tree buffers?

### DISCUSSION THREAD

**Discussion by bagnaram** [nvim-neo-tree/neo-tree.nvim#1194] [Q&A]
I want to disable the statusbar (the vanilla vim statusbar) inside of neo-tree buffers. Is this possible with `autocmd`? I have tried different combinations of:

```vimscript
autocmd BufEnter,BufWinEnter neo-tree* setlocal statusline=_

```

This works to varying degrees, but it is kind of a hack and requires focusing to the neotree buffer a few times.

**Comment by hsequeda** [comment]
I removed it from **Lualine** with the `disable_filetype option (lualine option).
